style.css

```diff
@@ -0,0 +1,503 @@
+  border: 1px solid #6070ff;
+  border-radius: 10px;
+  padding: 8px 16px;
+  font-weight: 500;
+  font-size: 17px;
+  color: #396df2;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.prjctbtn:hover {
+  background: #396df2;
+  color: #fff;
+}
+
+#about {
+  background-color: #fff;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-top-right-radius: 160px;
+}
+
+.abt-txt1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 50vw;
+  height: 50vh;
+  margin-left: 8pc;
+}
+
+.abt-txt2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 45vw;
+  height: 50vh;
+  margin-right: 5pc;
+}
+
+.abt-prmrytxt1 {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 30px;
+  margin-bottom: 2px;
+  color: #091e42;
+}
+
+.abt-prmrytxt2 {
+  font-weight: 400;
+  color: #344563;
+}
+
+.txt2 {
+  display: flex;
+  flex-direction: column;
+}
+
+.ctgry2 {
+  font-weight: 500;
+  font-size: 20px;
+  color: #000;
+}
+
+.ctgry3 {
+  padding: 10px;
+  font-size: 15px;
+  font-weight: 500;
+  color: #253858;
+}
+
+.lng-icn1 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 14px 6px;
+  width: 19pc;
+}
+
+.lnr-sprtr {
+  background-color: #dfe1e6;
+  width: 684px;
+  height: 4px;
+}
+
+.abtrw {
+  display: flex;
+  gap: 2px;
+  width: 0.1em;
+  height: 7pc;
+}
+
+.icn {
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding: 8px 16px;
+  list-style: none;
+  width: 122px;
+}
+
+#contact {
+  display: grid;
+  grid-template-columns: 30% 40% 30%;
+  background-image: url(./elements/cntctbckgrndpic.svg);
+  background-color: #6070ff;
+  border-top-left-radius: 120px;
+  width: 100vw;
+  height: 100vh;
+}
+
+.sct-1,
+.sct-2,
+.sct-3 {
+  background-image: inherit;
+}
+
+.sct-2 {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.cntct-ttl {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+}
+
+.cntct-info {
+  display: table-footer-group;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  align-content: center;
+  text-align: center;
+  width: 50%;
+  color: #ebebff;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  margin: 0.9em;
+}
+
+.frm-inputs {
+  display: contents;
+}
+
+.frm-tgs {
+  display: flex;
+  padding: 12px;
+  width: 80%;
+  font-size: 17px;
+  margin: 3%;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.sbmt-btn {
+  background: #fff;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+  padding: 6px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+  text-align: center;
+  font-size: 17px;
+  margin-left: 9px;
+  margin-bottom: 3rem;
+}
+
+.sbmt-btn:hover {
+  background: #7781e2;
+  color: #fff;
+}
+
+@media screen and (max-width: 768px) {
+  .links-section {
```

Comment:
- [ ] As mentioned by the previous reviewer, I think the problem is coming from here, I believe if you change this media query from max-width to `min-width`, it will consider your mobile design because your current page still does not have a responsive design yet 

![5](https://user-images.githubusercontent.com/77843050/213858322-57b02df8-ff20-4ac2-9160-5ec273d2f145.gif)